sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -725,9 +725,8 @@ trait CheckAnalysis extends PredicateHelper with LookupCatalog with QueryErrorsB
             val invalidExprSqls =
               PlanHelper.specialExpressionsInUnsupportedOperator(other).map(_.sql)
```

Comment:
Could you use `toSQLExpr()` instead of `sql()`

Reply:
Updated, thanks!